src/content/docs/fr/guides/typescript.mdx

```diff
@@ -169,20 +169,20 @@ const { greeting = 'Hello', name } = Astro.props;
 
 <p><Since v="1.6.0" /></p>
 
-Astro est livré avec des types utilitaires intégrés pour les types d'objets courants. Ils sont disponibles sous l'endpoint `astro/types`.
+Astro est livré avec quelques types d'utilitaires intégrés pour les modèles courants de typage de propriété. Ceux-ci sont disponibles sous le point d'entrée `astro/types`.
```

Comment:
I'm not proud of this translation... but the previous version was a bit wrong if we compare it with the original version (using `objets` when it's properties). If you have a better suggestion, I'm all ears!